docs/source/api_reference/layers.rst

```diff
@@ -166,6 +166,9 @@ Torch Layers
 .. autoclass:: deepchem.models.torch_models.InfoGraphEncoder
   :members:
 
+.. autoclass:: deepchem.models.torch_models.GINEncoder
```

Comment:
We should create a layers cheatsheet the way we have the models cheatsheet to track all the new layers. Could you take a crack at putting one together in a separate PR?